Userland/Games/GameOfLife/Pattern.cpp

```diff
@@ -24,13 +24,13 @@ void Pattern::set_action(GUI::Action* action)
 
 void Pattern::rotate_clockwise()
 {
-    Vector<DeprecatedString> rotated;
+    Vector<StringView> rotated;
     for (size_t i = 0; i < m_pattern.first().length(); i++) {
         StringBuilder builder;
         for (int j = m_pattern.size() - 1; j >= 0; j--) {
-            builder.append(m_pattern.at(j).substring(i, 1));
+            builder.append(m_pattern.at(j).substring_view(i, 1));
         }
-        rotated.append(builder.to_deprecated_string());
+        rotated.append(builder.string_view());
```

Comment:
This will lead to a use-after-free (and garbled patterns). Once that `StringBuilder` goes out of scope the StringView of it will be invalid.